cli/compose/loader/full-struct_test.go

```diff
@@ -887,3 +887,551 @@ x-nested:
 		workingDir,
 		workingDir)
 }
+
+func fullExampleJSON(workingDir string) string {
+	return fmt.Sprintf(`{
```

Comment:
This should be in a golden file IMHO.

Reply:
Well, there is some parameters (but we could "hard-code" them for the tests maybe)
The `yaml` one could be too indeed :stuck_out_tongue: 
(could be done in a followup for both)